js/login.js

```diff
@@ -21,8 +21,13 @@
 $(document).ready(function(){
 	var $loginMessage = $('#body-login').find('#message');
 	if ($loginMessage.length) {
-		var client = $("data[key='oauth2']").attr('value');
+		var client = $("data[key='oauth2']").attr('client');
 		var msg = t('oauth2', 'The application "{app}" is requesting access to your account. To authorize it, please log in first.', {app : client});
 		$loginMessage.parent().append('<div class="warning"><div class="icon-info-white" />'+msg+'</div>');
+		var user = $("data[key='oauth2']").attr('user');
+		if (user) {
+			$('#password').val('');
+			$('#user').val(user);
```

Comment:
@DeepDiver1975 what do you think of including:

```
$('#user').prop('readonly', true);
```

... so we force the user to login as the original user?

We'll loose focus on the login form though; password should receive it now for better UX I'm guessing.